atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -1791,7 +1791,7 @@ public void abort() {
                 ensureUncommitted();
                 if (state.compareAndSet(State.UNCOMMITTED, State.ABORTED)) {
                     if (hasWrites()) {
-                        preCommitRequirementValidator.throwIfPreCommitRequirementsNotMet(null, getStartTimestamp());
+                        preCommitRequirementValidator.throwIfPreCommitRequirementsNotMet(getStartTimestamp());
```

Comment:
In normal operation, the commit locks will be empty, so this is equivalent. In case of further features, where we'd add more locks are already present in the manager, we'd end up failing if any of those have been lost.

This however is also correct: for AdvisoryLocks, this method call will ALSO check that we haven't lost those locks. So this is all correct.

Reply:
Checked before and after and there are no functional changes.